experimenter/experimenter/targeting/constants.py

```diff
@@ -1067,6 +1067,19 @@ def __post_init__(self):
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+REVIEW_CHECKER_SIDEBAR_RECOMMENDATION = NimbusTargetingConfig(
+    name="Review Checker Sidebar Recommendation",
+    slug="review_checker_sidebar_recommendation",
+    description="Exclude users who have the Fakespot extension installed, "
+    "or who have the CFR pref set to false",
```

Comment:
Other targeting generally describe who is included, so this could be more along the lines of "Users who don't have Fakespot and allow feature recommendation'